docs/Components/Card.md

```diff
@@ -44,6 +57,6 @@ render(ExtensionPoint.MyExtension, () => <App />);
 
 | Name        | Type         | Description                                           | Required |
 | ----------- | ------------ | ----------------------------------------------------- | -------- |
-| content     | `string`     | Action label text.                                    | ☑️        |
+| content     | `string`     | Action label text.                                    | ☑️       |
```

Comment:
Why is the space gone? 🤔

Reply:
VSCode markdown auto format  is weird. It does similar thing to me too :D

Reply:
Ooh I see.. didn't realize these came from auto format!